quickstart.Dockerfile

```
@@ -1,44 +1,44 @@
 FROM docker.elastic.co/elasticsearch/elasticsearch:8.5.3
 
 # Environment variable
-ENV ARGILLA_LOCAL_AUTH_USERS_DB_FILE=/usr/share/elasticsearch/users.yml
-ENV TEAM_PASSWORD=1234
-ENV ARGILLA_PASSWORD=1234
-ENV TEAM_API_KEY=team.apikey
-ENV ARGILLA_API_KEY=argilla.apikey
+ENV ADMIN_USERNAME=admin
+ENV ADMIN_PASSWORD=11223344
+ENV ADMIN_API_KEY=team.apikey
```

Now that we are changing everything from team to admin this make sense, right?:

```suggestion
ENV ADMIN_API_KEY=admin.apikey
```